Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation3D.cxx

```diff
@@ -89,7 +89,7 @@ vtkSlicerMarkupsWidgetRepresentation3D::ControlPointsPipeline3D::ControlPointsPi
 
   this->SelectVisiblePoints = vtkSmartPointer<vtkSelectVisiblePoints>::New();
   this->SelectVisiblePoints->SetInputData(this->LabelControlPointsPolyData);
-  this->SelectVisiblePoints->SetTolerance(0.0); // we will set tolerance in world coordinate system
+  this->SelectVisiblePoints->SetTolerance(1e-4); // we will set tolerance in world coordinate system
```

Comment:
@lassoan  Look we will also need to update the comment.

Currently tolerance is set in world coordinate here:

https://github.com/Slicer/Slicer/blob/28bad9fd10e224c24d65853f1e0e92c3085a2582/Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation3D.cxx#L905

https://github.com/Slicer/Slicer/blob/28bad9fd10e224c24d65853f1e0e92c3085a2582/Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation3D.cxx#L1208

Reply:
@jcfr When looking into the issue, I found that the Z values were off by a minuscule difference and hence my change adds a tiny tolerance to offset floating point precision issues.

Reply:
@sankhesh should this be addressed in VTK proper as it seems to be a regression introduced by VTK9 ?  Or was VTK8 improperly working despite of us setting the tolerance to 0 ?

Reply:
@jcfr It doesn't look like a VTK issue and hence fixed at the application level.

Reply:
Great. Thanks for the clarification

Reply:
@lassoan  I would be curious to know what you think as you originally introduced https://github.com/Kitware/VTK/commit/c5ef1ee4ee3cb95dbd899a588c82c0a187a7566e